libvast/src/table_slice.cpp

```diff
@@ -282,6 +282,35 @@ void table_slice::offset(id offset) noexcept {
   offset_ = offset;
 }
 
+time table_slice::import_time() const noexcept {
+  auto f = detail::overload{
+    []() noexcept {
+      return time{};
+    },
+    [&](const auto& encoded) noexcept {
+      return state(encoded, state_)->import_time();
+    },
+  };
+  return visit(f, as_flatbuffer(chunk_));
+}
+
+void table_slice::import_time(time import_time) noexcept {
+  VAST_ASSERT(chunk_->unique());
```

Comment:
We can also "unsure" the chunk here, which is probably the safer option, and appears to be necessary to avoid larger unit test refactorings.